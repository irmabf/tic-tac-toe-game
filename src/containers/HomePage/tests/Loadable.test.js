import React from 'preact-compat'
import renderer from 'react-test-renderer'

import Loadable from '../Loadable'

describe('<Loadable />', () => {
	it('<Loadable /> renders correctly', () => {
		const tree = renderer.create(<Loadable />).toJSON()
		expect(tree).toMatchSnapshot()
	})
})
