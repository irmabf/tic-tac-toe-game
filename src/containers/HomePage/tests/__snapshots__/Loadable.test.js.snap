// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loadable /> <Loadable /> renders correctly 1`] = `
<div
  className="Wrapper-dyuznp iPelRH"
>
  <div
    className="Circle__CirclePrimitive-doUHun dKHgCb"
  />
  <div
    className="Circle__CirclePrimitive-iczIjT euoBoJ"
  />
  <div
    className="Circle__CirclePrimitive-cmIcdf dIjfue"
  />
  <div
    className="Circle__CirclePrimitive-jOiEYJ emKgyW"
  />
  <div
    className="Circle__CirclePrimitive-eAcCGr bEjFMn"
  />
  <div
    className="Circle__CirclePrimitive-fQPULp jJmoTs"
  />
  <div
    className="Circle__CirclePrimitive-egCDAQ jIAgSN"
  />
  <div
    className="Circle__CirclePrimitive-kwRWka iNRhgM"
  />
  <div
    className="Circle__CirclePrimitive-jnKZUF SVBFl"
  />
  <div
    className="Circle__CirclePrimitive-jSRngu iklYyO"
  />
  <div
    className="Circle__CirclePrimitive-gbjHSq iLEpFq"
  />
  <div
    className="Circle__CirclePrimitive-blqIGF ecSHje"
  />
</div>
`;
